tsconfig.json

```diff
@@ -11,11 +11,13 @@
 
     "declaration": true,
     "removeComments": true,
-    "target": "es2018",
-    "lib": ["es2018"],
+    "target": "es2020",
+    "lib": ["es2020", "DOM"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "outDir": "./lib"
+    "outDir": "./lib",
+    "jsx": "react",
+    "skipLibCheck": true
```

Comment:
Needed because if you import `compass-components` (or just some of the leafygreen things) this will blow up otherwise, the types depend on other packages’ types that aren’t available to dependencies (and I think it’s preferable to set this option here rather than to import reflux and storybook packages just for the types…)